jobmon_gui/src/workflow_overview_page/workflow_overview.tsx

```diff
@@ -119,6 +119,10 @@ function App() {
     setStatus(d["status"])
   });
 
+  const handleClear = handleSubmit((d) => {
+    navigate('/?user=' + "&tool=" + "&wf_name=" + "&wf_args=" + "&date_submitted=" + "&status=");
```

Comment:
Does this clear data from the page or reset the URL? If it's the former curious if it's easier to just reset state rather than issue a request

Reply:
This is reseting the URL which also clears out the user input in the form and clears out the workflow table.